chunky/src/res/se/llbit/chunky/ui/render/AdvancedTab.fxml

```diff
@@ -36,6 +36,8 @@
             <ChoiceBox fx:id="octreeImplementation" prefWidth="150.0" />
           </children>
         </HBox>
+        <IntegerAdjuster fx:id="gridSize" />
+        <CheckBox fx:id="preventNormalEmitterWithSampling" mnemonicParsing="false" text="Prevent normal emitter when using emitter sampling" />
```

Comment:
What exactly does this option do?

Reply:
It disables the usual contribution of emitters when emitter sampling is used as they produce noise when emitter sampling tries to not require a lot of SPP